articles/virtual-machines/extensions/azure-disk-enc-windows.md

```diff
@@ -24,15 +24,19 @@ Azure Disk Encryption leverages BitLocker to provide full disk encryption on Azu
 
 ## Prerequisites
 
-For a full list of prerequisites, see [Azure Disk Encryption for Linux VMs](../linux/disk-encryption-overview.md), specifically the following sections:
+For a full list of prerequisites, see [Azure Disk Encryption for Windows VMs](../windows/disk-encryption-overview.md), specifically the following sections:
 
-- [Azure Disk Encryption for Linux VMs](../windows/disk-encryption-overview.md#supported-vms-and-operating-systems)
+- [Supported VMs and operating systems](../windows/disk-encryption-overview.md#supported-vms-and-operating-systems)
 - [Networking requirements](../windows/disk-encryption-overview.md#networking-requirements)
 - [Group Policy requirements](../windows/disk-encryption-overview.md#group-policy-requirements)
 
 ## Extension schemata
 
-There are two schemata for the Windows AzureDiskEncryption extension: v2.2, a newer, recommended schema that does not use Azure Active Directory (AAD) properties, and v1.1, an older schema that requires AAD properties. You must use the schema version corresponding to the extension you are using: schema v2.2 for the AzureDiskEncryption extension version 2.2, schema v1.1 for the AzureDiskEncryption extension version 1.1.
+There are two versions of extension schema for Azure Disk Encryption (ADE):
+- v2.2 - A newer recommended schema that does not use Azure Active Directory (AAD) properties.
```

Comment:
This does not properly capture the prior information: 

Linux - AzureDiskEncryptionForLinux - is 0.1 (AAD) /1.1 (No-AAD)
Windows - AzureDiskEncryption is 1.1 (AAD) /2.2  (No-AAD)

Reply:
Hi @ejarvi,
I have just re-check content of both file (linux and windows) and versions seams to be aligned to your suggestion.

Reply:
Oops, this page is dedicated specifically to Windows now - so yes the alignment is correct.  👍